.github/workflows/scan_images.yaml

```diff
@@ -0,0 +1,21 @@
+name: Scan images
+
+on:  
+  schedule:
+    - cron: '00 23 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  on-push:
+    name: Get rocks modified and build-scan-test-publish them
```

Comment:
I think we could update those two fields since it's not running on push and doesn't do what the name says
```suggestion
  scan-images:
    name: Scan published images and report vulnerabilities
```